about.html

```diff
@@ -0,0 +1,94 @@
+    <title>About</title>
+    <link rel="stylesheet" href="https://use.typekit.net/vyw3zyb.css">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <header>
+        <div class="header1">
+            <ul class="header-links">
+                <li><img src="assets/images/facebook-icon.svg" alt=""></li>
+                <li><img src="assets/images/twitterIcon.png" alt=""></li>
+                <li>English</li>
+                <li>my page</li>
+                <li>log in</li>
+            </ul>
+        </div>
+        <nav>
+            <div class="logo-container"><a href="index.html"><img src="assets/images/logo.png" id="logo" alt="logo image"></a></div>
+            <img src="assets/images/menu-icon.svg" id="menu-icon" alt="hamburger menu icon" class="icon">
+            <ul class="nav-links" id="nav-links">
+              <img src="assets/images/menu-close-Icon.png" id="menu-close-icon" class="icon" alt="close menu icon">
+                <li><a href="index.html">Home</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="">program</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Join</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Sponsor</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">News</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Campaign</a></li>
+                <!-- <div class="divider"></div> -->
+            </ul>
+        </nav>
+      </header>
+      <main>
+        <section class="one">
+          <span class="sub-title">Hello! Music Producers!</span>
+          <h1 class="title" id="title">Tritone Music Producers conference 2023</h1>
+          <small class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur recusandae voluptatibus corporis ipsa temporibus minus incidunt deleniti in earum enim.</small>
+          <small class="contact-us">Please contact us via Email for any further questions about TMC production conference</small>
+          <a class="contact-link" href="#">Tritonemusic.co@enquiries.org</a>
+        </section>
+        <section class="two">
+            <h3>Tritone Music Co Producers Conference Logo</h2>
+            <div class="divider"></div>
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Odit labore saepe dolores dolorem laboriosam pariatur eveniet aliquid? Ipsum, architecto possimus.</p>
+            <figure class="logo-img"></figure>
+        </section>
+        <section class="three">
+            <h3>See the past TMC production Conference</h3>
+            <div class="divider"></div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore quisquam enim, atque nesciunt doloremque praesentium!</p>
```

Comment:
- [ ] As I can see this p here tag in your project is Lorem Ipsum but this placholders should be removed and put real information instead of it since it is one of the requirements to personalize the content of your page. Choose a topic that is different than the one in the original design. 🔥🔥 That's very important because this project you will add it to your Portfolio and the recruiters will see it in the future 💯✨

![image](https://user-images.githubusercontent.com/58553711/211713817-d7c8e44c-02b4-488e-bdc9-204d2a1a9402.png)